css/src/components/calendar.css

```diff
@@ -116,6 +116,14 @@
   background: var(--primary-lightest);
 }
 
+.Calendar-valueWrapper--inRange .Calendar-value:hover {
+  background: var(--primary-lighter);
```

Comment:
We can define a new class to handle this as we avoid using specificity CSS selectors. Please check once.